mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -483,18 +463,18 @@ def write_file(dsMOCTimeSeries):
             first = False
         date = days_to_datetime(ds.Time[tIndex], calendar=calendar)
         print '     date: {:04d}-{:02d}'.format(date.year, date.month)
-        horizontalVel = ds.avgNormalVelocity[tIndex, :, :].values
-        verticalVel = ds.avgVertVelocityTop[tIndex, :, :].values
-        velArea = verticalVel * areaCell[:, np.newaxis]
-        transportZ = _compute_transport(maxEdgesInTransect,
-                                        transectEdgeGlobalIDs,
-                                        transectEdgeMaskSigns,
+        # Cache results if their size is reasonable
+        load = partial(preload, maxgb=config.getfloat('input',
+                                                      'reasonableArraySizeGB'))
+        horizontalVel = load(ds.avgNormalVelocity.isel(Time=tIndex))
+        velArea = load(ds.avgVertVelocityTop.isel(Time=tIndex) * areaCell)
```

Comment:
Wouldn't it actually be clearer to read if you just cached the value of `reasonableArraySizeGB` and had 2 calls to `preload` without the need to use `partial`?  If we want future developers to use `preload`, I think we want it to be clear how to use it.